api/server.js

```diff
@@ -1,19 +1,23 @@
 // Entry point for the API
 
 import express from 'express';
-import routes from './src/routes/index.js';
+import cors from 'cors';
+import cvRoutes from './src/routes/cvRoute.js';
+import aiEnhance from './src/routes/aiEnhanceRoute.js';
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 // Middleware
 app.use(express.json());
+app.use(cors({ origin: 'http://localhost:5173', credentials: true }));
 
 // HINT: Add authentication middleware here if needed
 // Example: app.use(authMiddleware);
 
 // API Routes
-app.use('/api', routes);
+app.use('/generateCv', cvRoutes);
+app.use('/aiEnhance', aiEnhance);
```

Comment:
Consider organizing route paths more RESTfully:
 Instead of these paths it might be more simple to use
```js
 app.use('/api/cv', cvRoutes);
 app.use('/api/cv/enhance', aiEnhance);
```